virttest/utils_conn.py

```diff
@@ -582,7 +582,7 @@ def conn_recover(self):
 
         logging.debug("SSH authentication recover successfully.")
 
-    def conn_setup(self):
+    def conn_setup(self, rTimeout=10):
```

Comment:
Yes, we use all lower case with underscores instead of camel case, so I'd ask you to convert this to `r_timeout` or similar.